src/Symfony/Component/HttpFoundation/Session/Session.php

```diff
@@ -29,6 +29,7 @@ class Session implements SessionInterface, \IteratorAggregate, \Countable
     private $flashName;
     private $attributeName;
     private $data = array();
+    private $wasStarted;
```

Comment:
Can we rename the property and the method to `hasBeenStarted` instead of `wasStarted`? It feels to me that `was` implies it isn't "started" anymore.

Reply:
renamed